module/Photo/src/Service/Photo.php

```diff
@@ -575,6 +575,7 @@ private function storeProfilePhoto(
      */
     public function setProfilePhoto(int $photoId): void
     {
+        // TODO: Add null check for `$photo`.
```

Comment:
This TODO mark should either be addressed or removed if it is not necessary.

Reply:
It should definitely be done at some point, but I don't want to do it now. I'll make an issue for it and remove it from this PR.

Reply:
Alright, sounds good too.